### PR TITLE
[Style/#89]:구매자 상세페이지 UI

### PR DESCRIPTION
**Pull request by juhee067**
## 📌 작업 내용

- 구매자 상세페이지 테이블 생성
- 전화번호 형식 000-0000-000 변환 함수
- 날짜 데이터 YYYY-MM-DD 변환 함수

<br/>

## 📌 구현 결과 (선택)
**[구매자 상세 조회]**
image
<br/>
## 📌 기타사항
- 주소는 api 작업 후 수정

### CODE REVIEW

**Comment by JIMIN1020** [comment]
<h2>Coverage Report</h2>
<table> <thead> <tr> <th align="center">Status</th> <th align="left">Category</th> <th align="right">Percentage</th> <th align="right">Covered / Total</th> </tr> </thead> <tbody> <tr> <td align="center">🔵</td> <td align="left">Lines</td> <td align="right">9.03%</td> <td align="right">540 / 5977</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Statements</td> <td align="right">9.03%</td> <td align="right">540 / 5977</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Functions</td> <td align="right">16.66%</td> <td align="right">41 / 246</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Branches</td> <td align="right">26.95%</td> <td align="right">76 / 282</td> </tr> </tbody> </table><details><summary>File Coverage</summary><table> <thead> <tr> <th align="left">File</th> <th align="right">Stmts</th> <th align="right">% Branch</th> <th align="right">% Funcs</th> <th align="right">% Lines</th> <th align="left">Uncovered Lines</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6"><b>Changed Files</b></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx">src/components/Liquor/GroupBuyingCard.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L7-L10" class="text-red">7-10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L12-L14" class="text-red">12-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L16-L22" class="text-red">16-22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L24-L32" class="text-red">24-32</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L34" class="text-red">34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L36" class="text-red">36</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L38" class="text-red">38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L45" class="text-red">45</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L49-L50" class="text-red">49-50</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L52" class="text-red">52</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L55-L57" class="text-red">55-57</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L60" class="text-red">60</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L68" class="text-red">68</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L71" class="text-red">71</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/components/Liquor/GroupBuyingCard.tsx#L87-L89" class="text-red">87-89</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx">src/features/Catalog/components/Filter/FilterOption.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L12-L19" class="text-red">12-19</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L21-L23" class="text-red">21-23</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L25-L33" class="text-red">25-33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L35" class="text-red">35</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L37-L46" class="text-red">37-46</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L48-L54" class="text-red">48-54</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L56" class="text-red">56</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L58-L61" class="text-red">58-61</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L63" class="text-red">63</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L65" class="text-red">65</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L67-L68" class="text-red">67-68</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L71" class="text-red">71</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L79-L80" class="text-red">79-80</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L83-L84" class="text-red">83-84</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L87" class="text-red">87</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L95-L97" class="text-red">95-97</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Catalog/components/Filter/FilterOption.tsx#L99" class="text-red">99</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx">src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L1-L5" class="text-red">1-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L8-L11" class="text-red">8-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L13-L16" class="text-red">13-16</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L18-L24" class="text-red">18-24</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L26" class="text-red">26</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L28" class="text-red">28</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/GroupBuying/components/GroupBuyingGrid/GroupBuyingGrid.tsx#L30" class="text-red">30</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/api/signup.api.ts">src/features/Join/api/signup.api.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/api/signup.api.ts#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/api/signup.api.ts#L5" class="text-red">5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/api/signup.api.ts#L7" class="text-red">7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/api/signup.api.ts#L9-L11" class="text-red">9-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/api/signup.api.ts#L13-L16" class="text-red">13-16</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/api/signup.api.ts#L18-L25" class="text-red">18-25</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/api/signup.api.ts#L27-L36" class="text-red">27-36</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx">src/features/Join/components/IndustryForm.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L1-L10" class="text-red">1-10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L12-L15" class="text-red">12-15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L17" class="text-red">17</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L19-L20" class="text-red">19-20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L22-L42" class="text-red">22-42</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L44-L48" class="text-red">44-48</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L50-L64" class="text-red">50-64</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L66" class="text-red">66</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L68" class="text-red">68</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L70" class="text-red">70</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L77" class="text-red">77</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L89" class="text-red">89</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L93" class="text-red">93</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L98" class="text-red">98</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/IndustryForm.tsx#L100" class="text-red">100</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx">src/features/Join/components/PasswordWithIdForm.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L1-L7" class="text-red">1-7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L9-L13" class="text-red">9-13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L15-L20" class="text-red">15-20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L22-L34" class="text-red">22-34</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L37-L42" class="text-red">37-42</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L44-L46" class="text-red">44-46</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L48-L51" class="text-red">48-51</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L53-L55" class="text-red">53-55</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L57" class="text-red">57</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L59" class="text-red">59</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L61" class="text-red">61</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L68" class="text-red">68</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L74" class="text-red">74</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L79" class="text-red">79</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Join/components/PasswordWithIdForm.tsx#L81-L82" class="text-red">81-82</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyDetail.tsx">src/features/Management/components/BuyerDetail/BuyDetail.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L6-L17" class="text-red">6-17</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L19-L20" class="text-red">19-20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L22" class="text-red">22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L24" class="text-red">24</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx">src/features/Management/components/BuyerDetail/BuyerTable.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L4-L14" class="text-red">4-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L16-L39" class="text-red">16-39</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L41-L53" class="text-red">41-53</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L55-L61" class="text-red">55-61</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L63" class="text-red">63</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L65" class="text-red">65</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L67" class="text-red">67</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L77" class="text-red">77</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L82-L83" class="text-red">82-83</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L89" class="text-red">89</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L94-L95" class="text-red">94-95</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L102-L104" class="text-red">102-104</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L109-L110" class="text-red">109-110</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/pages/BuyerDetailPage.tsx">src/pages/BuyerDetailPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/pages/BuyerDetailPage.tsx#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/pages/BuyerDetailPage.tsx#L3-L7" class="text-red">3-7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/pages/BuyerDetailPage.tsx#L9" class="text-red">9</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/pages/BuyerDetailPage.tsx#L11" class="text-red">11</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/routes/index.tsx">src/routes/index.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/routes/index.tsx#L1-L17" class="text-red">1-17</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/routes/index.tsx#L19-L25" class="text-red">19-25</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/routes/index.tsx#L27-L53" class="text-red">27-53</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/routes/index.tsx#L55-L75" class="text-red">55-75</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/routes/index.tsx#L77-L94" class="text-red">77-94</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/routes/index.tsx#L96-L98" class="text-red">96-98</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/routes/index.tsx#L100" class="text-red">100</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/utils/format.ts">src/utils/format.ts</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/utils/format.ts#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/utils/format.ts#L5-L7" class="text-red">5-7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/c7e30eef062cf45a0ee38b8fcf92db4fa2093272/src/utils/format.ts#L9-L11" class="text-red">9-11</a></td> </tr> </tbody> </table></details>
<em>Generated in workflow <a href=https://github.com/Licruit/Licruit-frontend/actions/runs/10505683523>#140</a></em>

File: src/utils/format.ts [hunk 1]
```diff
@@ -1,3 +1,11 @@
 export const formatNumber = (value: number) => {
   return value.toLocaleString();
 };
+
+export const formatDate = (date: string) => {
+  return new Date(date).toLocaleDateString();
```
Comment (JIMIN1020) [inline src/utils/format.ts:6]: dayjs 사용 중이니 그걸로 대체해도 될듯요!

**Review by JIMIN1020 — APPROVED** [review]
주희님 제가 맡았던 파트까지 해주시고..
고생 많으셨습니다~!!

File: src/features/Management/components/BuyerDetail/BuyerTable.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    totalPrice: 70000,
+  };
+
+  return (
+    <Container>
+      <table>
+        <thead>
+          <tr>
+            <th scope='col' colSpan={2}>
+              신청 상세내역 <span>{formatDate(mockData.createdAt)}신청</span>
```
Comment (SebellKo) [inline src/features/Management/components/BuyerDetail/BuyerTable.tsx:22]: 리액트 데이트피커 사용하면서 해당 날짜 형식 변경하기 위해서 date-fns 라이브러리 사용중이긴 한데 date-fns 라이브러리의 format 메소드 사용해도 좋을거 같아요 !!

**Review by SebellKo — APPROVED** [review]
주히님 고생고생이요 ~~

**Pull request merged**